### PR TITLE
Re-enable ownKeys.list and ownKeys.register

### PR DESCRIPTION
**Pull request by Powersource**
Fixes https://github.com/ssbc/ssb-tribes/issues/100

* [x] re-enable
* [x] check readme. maybe mention that the list is always 1 long
* [x] add test

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
     "cross-env": "^7.0.3",
     "is-canonical-base64": "^1.1.1",
     "scuttle-testbot": "^2.1.0",
-    "ssb-box2": "^7.2.0",
+    "ssb-box2": "github:ssbc/ssb-box2#get-own-dm",
```
Comment (Powersource) [inline package.json:46]: https://github.com/ssbc/ssb-box2/pull/37

File: README.md [hunk 2]
```diff
@@ -88,7 +88,7 @@ A Secret-Stack server running the plugins:
 - `ssb-tribes`
 - `ssb-db2/compat`
 - `ssb-db2/compat/feedstate`
-- `ssb-box2` >= 7.2.0
+- `ssb-box2` >= TODO
```
Comment (mixmix) [inline README.md:91]: you said it

File: index.js [hunk 3]
```diff
@@ -371,24 +371,20 @@ function init (ssb, config) {
...
-    //  }
-    // }
+    ownKeys: {
+      list (cb) {
+        ssb.box2.getOwnDMKey((err, dmKeyInfo) => {
+          if (err) return cb(Error("Couldn't get own dm key on ownKeys.list", { cause: err }))
+          return cb(null, [dmKeyInfo])
+        })
+      },
+      register (key) {
```
Comment (mixmix) [inline index.js:384]: may be good to call it set .. as it's not just registering, but rather setting the one you will use

**Review by mixmix — APPROVED** [review]

File: index.js (hunk 3 above)
Comment (Powersource) [reply index.js:384]: yeah agree but i think there's so many breaking changes atm, i think it's better to do this at some later point

**Pull request merged**